src/tools/crawler/main.rs

```diff
@@ -75,33 +75,68 @@ fn start_logger(default_level: LevelFilter) {
         .init();
 }
 
+/// Parses and converts `String` values found in a `Vec` to valid `SocketAddr`.
+///
+/// # Input
+///
+/// Valid inputs can be in the following forms:
+/// - IP + port (both IPv4 and IPv6 are valid)
+/// - IP (Default `ZCASH_P2P_DEFAULT_PORT` will be appended)
+/// - DNS + port
+/// - DNS
+fn parse_addrs(seed_addrs: Vec<String>, crawler: &Crawler) -> Vec<SocketAddr> {
```

Comment:
a unit test could be written for something like this - we should not forget to do that sometimes, from time to time 😃

Reply:
I'll add some tests for the parsing of standard IP addresses for now. Not sure if we could add tests for hostname resolution since that depends on external factors :thinking:

Reply:
Done